### PR TITLE
Allow assessment archiving from assessments list

### PR DESCRIPTION
**Pull request by derekprior**
Previously this functionality was only accessible via the results
screen.

### CODE REVIEW

**Comment by derekprior** [comment]
abet_outcomes_assessment

@alexandra-aas I'm getting wrapping in the table now. Any ideas for an easy fix? I tried "whitespace: nowrap" on the actions div and that fixes that column but the others wrap instead. Do you have any ideas?

Wonder if maybe we can live without the target percentage and minimum grade columns right there? What do you think @jdaries ?

**Comment by alexandra-aas** [comment]
@derekprior I'll take a look. We could also just switch back to not inlining them and having them stacked again. My intention was to make the rows less tall.

**Comment by jdaries** [comment]
@derekprior i would be okay with eliminating those two columns

File: app/views/manage_assessments/assessments/_direct_assessments.html.erb [hunk 1]
```diff
@@ -13,11 +13,17 @@
     <td><%= assessment.minimum_requirement %></td>
     <td><%= assessment.target_percentage %></td>
     <td class="inline-action-links">
```
Comment (alexandra-aas) [inline app/views/manage_assessments/assessments/_direct_assessments.html.erb]: You can do: `<td class="inline-action-links" nowrap>`.

**Comment by alexandra-aas** [comment]
@derekprior The above fixes the issue, but I would also be for eliminating those two rows because I don't think that information is crucial at that level.

One other thing. I added an archive button to `manage_results/subjects/show.html.erb` in my last PR. Do you want to double check that I have the right code there considering these changes.

**Comment by derekprior** [comment]
I'll fix this up and take a look at that, yes. Thanks

**Pull request merged**